README.md

```diff
@@ -101,14 +132,102 @@ credentials, ie:
 }
 ```
 
-## Publishing
+The contents of this file can be found in lastpass with the name
```

Comment:
I meant merely something like: "On Azure, to regenerate this file, go to this link: http://some.awful.site.on.portal.azure.com/"